modules/ROOT/pages/index.adoc

```diff
@@ -185,6 +185,32 @@ Anypoint Monitoring is the standard method of monitoring Mule application and AP
 
 |===
 
+== Use Cases
+
+The following use cases provide examples of how you can use Anypoint Monitoring.
+
+* Understand performance-related KPIs+
+DevOps engineer Gary maintains a series of APIs related to a transaction processing system. Because the TPS requires swift response time, *Gary uses API response time dashboards and alerts to send an email when there is a bottleneck*. After gathering some data, Gary can then size his applications.
```

Comment:
Is there any reason for using bold formatting in this scenario: "**Gary uses API**..." but not in the others?

Reply:
Unintentional - thank you

Reply:
```suggestion
DevOps engineer Gary maintains a series of APIs related to a transaction processing system. Because the TPS requires swift response time, Gary uses API response time dashboards and alerts to send an email when there is a bottleneck. After gathering some data, Gary can then size his applications.
```